README.md

```
@@ -1,95 +1,303 @@
+-   **Python**: Version 3.8 or higher (Python 3.8.13 is used in the Docker image).
+-   **System Packages**: Install `libsqlcipher-dev` and `libsqlite3-dev` (or equivalent for your OS).
     ```sh
     sudo apt install libsqlcipher-dev libsqlite3-dev
     ```
-- Initialize virtualenv:
-    ```sh
-    virtualenv venv_token_seeder
-    ```
-- Edit the `activate` file of the virtualenv (`venv_token_seeder/bin/activate`), and add the following lines in the end of the file.
+-   **Virtual Environment**:
+    -   Initialize: `virtualenv venv_token_seeder`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)